volumes/rescu/README.md

```diff
@@ -1,45 +1,316 @@
+### 4- How often is the data updated?	
+>Raw data feeds into oracle every 20 seconds. Data (Jim's aggregation) pushed into Oracle daily. Data Pipeline Flow and Transformation – 
+## **Methods for processing the data: Data Pipeline Flow and Transformation- How does the data move/transform through the organization?**
+	
+### 1- How is the data organized and aggregated? 	
+> Raw data is recorded in 20 second increments. Data handled by D&A is aggregated to 15 minute increments and produced as reports out of Oracle. Data is matched to arterycode (directional).Loop detectors, radar detectors have coordinates. 
+
+### 2- Where is the raw data stored?	   	
+>Raw data is stored in Oracle and ITS Central. 
+Within ITS Central it is stored in Postgres database but we are unsure as to how much data is being stored in the system.
+### 3- How is the raw data stored?	
+> Unsure—particularly now that there are two streams (one in ITS Central and one in Oracle).
+
+### 4- Where is the data stored?	
+>Pulling data from databases into Postgres. Windows task scheduler is responsible for pulling the data from \\tssrv. 
+>Refer to [readme_pull](https://github.com/CityofToronto/bdit_data-sources/blob/master/volumes/rescu/README_PULL.md) for more information
+
+### 5- How is the data stored?	
+>Unsure.   
+
+
+
+
+## Instrument/device or software-specific information needed to interpret the data:
+> Provide information on how data can be interpreted baed on collection device/instrument  
+
+
+
+## Standards information, if appropriate: 
+> N/A- to be added
+
+
+
+## Environmental/experimental conditions: 
+> N/A- to be added 
+
+
+
+## Data Quality/ Describe any quality-assurance procedures performed on the data
+
+### 1- Are there known data gaps/incomplete data? 	
+>Yes, there are many data gaps.
+### 2- What are they gaps?	
+>Missing volumes due to detector issues. Data reports sent to D&A out of Oracle contain specific lane-level data. This may help isolate detectors that are down and help in closing gaps and validating data.
+
+### 3- How are data gaps/incomplete data addressed? 
+>Currently within D&A and for the purpose of Data Requests, requesters tell requestees that data is not available.  
+
+### 3- Who is responsible for addressing data gaps/incomplete data?
+>Unsure. By default it is D&A.
+
+#### 4- Are there data quality assessment processes for the data?
+>Yes, there is a pipeline check but it regularly fails so it may not a sufficient form of quality assessment.No upstream validation processes identified.
+
+#### 5- How often are data quality assessment processes for the data undertaken? 
+>D&A process done daily. QA process counts the number of rows that have data with 7000 rows being the threshold.   
+ 
+#### 6- Are external contractors or consultants responsible for checking data quality?
+> Unsure. 
+
+
+## Data Validity- Is the data truly representative of the real world? 
+	
```

Comment:
I feel like everything we know about data validity / data quality has already been said...